### ISSUE TITLE
New help page section on when it is fair to request others' personal information

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/whatdotheyknow-theme#1054]
Possibly in the form of a new question:

>Can I request information which relates to another person?

at

https://www.whatdotheyknow.com/help/requesting

Arises from #1050

**Comment by RichardTaylor** [comment]
Asking users to only make requests for other people's personal information when it is fair to do so is a good summary of the position, however users may need more guidance than that. 

A key principle is already in the house rules: "Only use WhatDoTheyKnow.com to request information which anyone could expect to obtain if they requested it" What we want to do here is explain what that means in terms of others' personal information, and when one can reasonably request, and expect to receive, others' personal information. 

We also have factors relating to what we are likely to be able to justify publishing if challenged. We don't want users making requests which we can't justify publishing. We already have related house rules including: "Do not include potentially defamatory/potential libellous comments (such as allegations) in your requests and annotations".  

We're not summarising the law, but giving our own guidance, which may form part of our own house rules for users of WhatDoTheyKnow. 

Proposed text:

=====

**Can I request information which relates to another person?**
     
Unless it is fair to do so you must not use our service to request other people’s personal information or include others' personal information in requests, annotations or follow-ups. This is set our in our[ house rules](https://www.whatdotheyknow.com/help/house_rules). 

Requests for others' personal information are likely to be rejected if it releasing the information in question is not considered to be fair and we only permit requests where there is a reasonable expectation of the release of the material sought. [The Information Commissioner has published detailed guidance on when personal information should be released in response to a request for information under the Freedom of Information Act or Environmental Information Regulations.](https://ico.org.uk/media/for-organisations/documents/2614720/personal-information-section-40-and-regulation-13-version-21.pdf) 

Generally it is less likely to be fair to request information about the private elements of other people's lives such as their health, than it is about public, civic, aspects of their lives. 

Requests relating to individuals' roles as, for example, elected representatives, or regulated professionals (police constables, teachers, doctors, social workers etc.) are often considered to be fair. 

It is important when making requests which relate to other people not to include any potentially defamatory comments such as allegations in your correspondence. Again this is covered by our [house rules](https://www.whatdotheyknow.com/help/house_rules) which apply to all use of our service. 

 =====

Other words we could use in addition to fair: legal, lawful, necessary, justified, legitimate.

**Comment by mdeuk** [comment]
I would +1 this; however, I think there is an accessibility challenge we need to address.

From a readability perspective, I think this current proposal might be too complex. Can we simplify it? This could be related to #987 - and if we had a "widening access" tag, I'd be inclined to add it.

### Rationale

The text proposed scores **35.6** on the [Flesch Reading Ease](https://en.wikipedia.org/wiki/Flesch%E2%80%93Kincaid_readability_tests) scale, which puts it into the "difficult to read" territory.

<img src="https://user-images.githubusercontent.com/249418/158096493-b3c4db75-1132-4da2-9a60-f2e00bf08f36.png" width="50%">

Ideally, we want to aim for a score of 60+ if we can. That isn't always viable, of course; but at the moment, if we went ahead with this proposal 'as is', we could make it quite difficult for some of our users to interpret _what we mean_.

### Why does this matter?

We need to ensure that, wherever possible, our communications are as inclusive as possible. Many of our users are not native english speakers, and anecdotal evidence suggests that some users do not necessarily understand some of our comms.

We also have to consider that people who have learning difficulties, [SpLD](https://www.kent.ac.uk/teaching/networks/ltn/documents/2014-15/18-Sept-2014-SpLD-handout-L-Regan.pdf)s, and / or neurodiverse conditions might have difficulty in following text which is overly complex.

The concept of [readability](https://en.wikipedia.org/wiki/Readability) applies here - as does [plain english](http://www.plainenglish.co.uk/). Admittedly, I'm not great at this myself!

### Resources

There's some really helpful guidance on the ["Writing for gov.uk"](https://www.gov.uk/guidance/content-design/writing-for-gov-uk) manual (produced by GDS) which could help us to an extent. This is based on "tried and tested" research used by Government agencies (in a UK specific format). We also, of course, have the mySociety style guide!

There's also a [list of tools](https://style.ons.gov.uk/category/writing-for-the-web/how-we-read-on-the-web-writing-for-the-web/) that the ONS have put together, amongst others, which give resources that can be used to help us assess our content.

Of course, scores are just a tool to help assess how readable something is - but they can be misleading, as you can get a decent score without having content that makes much sense. The style guide may help us refine how we present things, but proof reading is vital! 😃 

### Possible way forward

One solution here could be to write a **summary** of the rules, and then link to the expanded version which outlines precisely what we mean. Call it a [TL;DR](https://en.wikipedia.org/wiki/Wikipedia:Too_long;_didn%27t_read) version perhaps.

My reason for linking to #987 is broadly similar - it's a wordy document, and it isn't always the easiest to read. We've incrementally added things over a number of years, but we need to ensure good readability. The two documents interact with each other in a number of ways, so it'd be worthwhile tackling both of these issues.

**Comment by RichardTaylor** [comment]
I agree our advice should be as accessible and easy to read as possible. 

We could:
* Make minor tweaks to use shorter sentences
* Include examples of what is, and is not, acceptable

**Comment by RichardTaylor** [comment]
We could usefully add:

>Only refer to, or request, others' personal information if it is necessary. Where possible try to depersonalise your correspondence. 

Sometimes users appear to name individuals in requests as part of a course of harassment, or in an attempt to link individuals to particular subjects in search results.  We could seek to specifically address this in this advice.

**Comment by mdeuk** [comment]
Adding a link to the Plain English campaign's "How to" document: https://www.plainenglish.co.uk/files/howto.pdf

**Comment by HelenWDTK** [comment]
This issue is being closed due to a lack of discussion or resolution for over 12 months. Should we decide to revisit this issue in the future, it can be reopened.
